data/json/recipes/recipe_others.json

```diff
@@ -1620,6 +1631,17 @@
       [ [ "stick", 2 ], [ "broom", 2 ], [ "2x4", 2 ], [ "pool_cue", 2 ] ]
     ]
   },
+  {
+    "type": "recipe",
+    "result": "leg_xlsplint",
+    "copy-from": "leg_splint",
+    "time": 2000,
+    "components": [
+      [ [ "rag", 10 ], [ "medical_gauze", 4 ] ],
```

Comment:
Most of the ingredients are the same as the original human-sized leg splint, so you might want to update these; potentially even switch stick ("heavy stick") to stick_long ("long stick") or some other way.